### PR TITLE
https://issues.redhat.com/browse/ACM-16029

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-16029

### CODE REVIEW

File: clusters/release_notes/mce_errata.adoc [hunk 1]
```diff
@@ -7,16 +7,19 @@ If no release notes are listed, the product does not have an Errata release at t
 
 *Important:* For reference, Jira links and Jira numbers might be added to the content and used internally. Links that require access might not be available for the user. 
 
-== Errata 2.7.1
+== Errata 2.6.4
 
 * Delivers updates to one or more product container images.
 
-* Fixes an error with the *Clear all filters* button. (https://issues.redhat.com/browse/ACM-15277[ACM-15277])
+* Stops the `Detach clusters` action from deleting hosted clusters. (https://issues.redhat.com/browse/ACM-15058[ACM-15058])
```
Comment (jc-berger) [inline clusters/release_notes/mce_errata.adoc:14]: @rhaignere like you say in the Jira, these issues have been resolved in earlier versions. 

However, I see there are distinct Jiras for these same issues for the 2.7.1 version. 

Do you want there to be duplication of these fixes for both the Errata 2.6.4 and 2.7.1? 

Or are you fine with what I did here: remove the content from the 2.7.1 errata and put it in the 2.6.4 errata?

Comment (rhaignere) [reply clusters/release_notes/mce_errata.adoc:14]: Hi @jc-berger - I would prefer the duplication of these fixes for both the Errata for 2.6.4 and 2.7.1, unless this is confusing. Thank you!

**Review by rhaignere — APPROVED** [review]
lgtm
thanks for the quick updates!

**Review by swopebe — APPROVED** [review]
Looks great. TY!

**Pull request merged**